contracts/Otoken.sol

```diff
@@ -139,12 +139,13 @@ contract Otoken is ERC20Initializable {
             )
         );
 
-        // concatenated symbol string: oETHUSDC-05SEP20-200P
+        // concatenated symbol string: oETHUSDCUSDC-05SEP20-200P
```

Comment:
did we also want to have the name use / instead of -? :D

Reply:
@mshrieve @aleone

Reply:
Yes will add it.

Reply:
I think the suggestion was to add the `/` before the collateral asset.